readme.md

```diff
@@ -721,6 +721,8 @@
 | [Headless UI](https://headlessui.dev/)| Completely unstyled, fully accessible UI components, designed to integrate beautifully with Tailwind CSS. |
 | [Styled components](https://styled-components.com/)| Build beautifully UI Components, for your applications & websites. |
 | [Figma Bootstrap 5 UI Kit](https://drive.google.com/file/d/1SZOTxXYfCMujqP2P684Vd5QIrlk5bwe6/view?usp=sharing)| UI Kit comprising 300+ organized Bootstrap 5 components built with atomic design system & auto layout. |
+| [CSS Scan](https://getcssscan.com/css-buttons-examples)| 84 Beautiful CSS buttons examples. |
+| [CSS Scan](https://getcssscan.com/css-box-shadow-examples)| 91 Beautiful CSS box-shadow examples. |
```

Comment:
Please move the links to the inspiration section as it's not 100% UI Components, 
also, the name of the resource should not repeat itself, please add a more descriptive name `[Css buttons]` etc